### ISSUE TITLE
Compact Forms: Add more settings for general compactness/density

### ISSUE THREAD

**Issue by tmcconechy** [infor-design/enterprise#3744]
**Is your feature request related to a problem or use case? Please describe.**

We are getting more and more requests for either more compact forms or forms with larger fonts that fill more vertical space. This can add accessibility value and accommodate more teams.

**Describe the solution you'd like**

- Introduce more compactness settings (Compact, Cosy, Default, Large)
- Reconsider if the default should be smaller (Cosy?) as that is the bigger request
- Ability to set the density (space between items and size)
- Ability to set the font size 
- Make this a user/app preference
- Apply these settings to everything (Forms, Lists, Buttons, Datagrid ect)

**Describe alternatives you've considered**
Some less combination of the above. This may introduce a LOT of bugs.

**Additional context**
https://github.com/infor-design/enterprise/issues/3742
Email: Infor PathWay
Landmark (Koch)
Study the density options in material https://material-ui.com/customization/density/#applying-density they solve similar cases

**Comment by tmcconechy** [comment]
This has been completed
